app/controllers/api/greetings_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::GreetingsController < ApplicationController
+  def random
+    random_greeting = Message.order('RANDOM()').first
```

Comment:
- Two apps connect together successfully and random greetings are display :rocket: 
![Screenshot from 2023-09-26 10-28-48](https://github.com/iamsjunaid/hello-rails-react/assets/63087491/9f830428-0ee2-4f65-951c-91af70816af0)